CHANGELOG.md

```diff
@@ -27,6 +27,19 @@ Ref: https://keepachangelog.com/en/1.0.0/
 -->
 
 # Changelog
+## v5.7.1 & v5.7.2
+sei-chain
```

Comment:
Shall we add changelog for switching evm receipt state to its own receipt.db?

Reply:
that was in v5.6 versions i believe https://github.com/sei-protocol/sei-chain/blob/main/CHANGELOG.md#v560